packages/common/src/core/slickGrid.ts

```diff
@@ -593,8 +593,7 @@ export class SlickGrid<TData = any, C extends Column<TData> = Column<TData>, O e
         if (emptyTarget) {
           emptyElement(target);
         }
-        const node = options?.cloneNode ? val.cloneNode(true) : val;
```

Comment:
could you also remove the options prop above `cloneNode?: boolean;`  since it's no longer needed